src/commands/lambda/flare.ts

```diff
@@ -390,9 +455,17 @@ export const getLogStreamNames = async (cwlClient: CloudWatchLogsClient, logGrou
   const output: string[] = []
   for (const logStream of logStreams) {
     const logStreamName = logStream.logStreamName
-    if (logStreamName) {
-      output.push(logStreamName)
+    if (!logStreamName) {
+      continue
+    }
+    if (rangeSpecified) {
+      const lastEventTime = logStream.lastEventTimestamp
+      const firstEventTime = logStream.firstEventTimestamp
+      if (lastEventTime && firstEventTime && (lastEventTime < startMillis || firstEventTime > endMillis)) {
```

Comment:
Is there a way to improve the readability of this condition?

Reply:
Changed to:
```
if (rangeSpecified) {
  const firstEventTime = logStream.firstEventTimestamp
  const lastEventTime = logStream.lastEventTimestamp
  if (lastEventTime && lastEventTime < startMillis) {
    continue
  }
  if (firstEventTime && firstEventTime > endMillis) {
    continue
  }
}
```

What do you think?